client/config/deploymentSpecificConfig.js

```diff
@@ -21,3 +21,5 @@ tieConfig.constant('SERVER_URL', null);
 tieConfig.constant('PRIMER_DIRECTORY_URL', 'docs/');
 
 tieConfig.constant('ALLOW_PRINTING', false);
+
+tieConfig.constant('PARENT_PAGE_ORIGIN', null);
```

Comment:
PARENT_PAGE_URL? Don't know what an "origin" is.

Reply:
Redefined to be PARENT_PAGE_URL_ORIGIN as the constant is only the URL origin rather than the entire URL (more info at https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy).